### DISCUSSION TITLE
Generating clustered orientations

### DISCUSSION THREAD

**Discussion by JasonOttUW** [mtex-toolbox/mtex#967] [Ask Anything] [answered]
Hello everyone!

I am working on a project involving seismic anisotropy, and want to use MTEX to generate figures showing how the seismic anisotropy of a single mineral is related to the strength of the orientations of the grains of the mineral for early in my talk to a general audience. I understand how to generate a completely random set of orientations, but how would I go about generating a moderately oriented set of orientations (for example, the {100} planes clustered around the Z-direction, {001} planes clustered about Y, and the {001} clustered about X?

**Comment by kilir** [comment, marked as answer]
Hi,
have a look at `unimodalODF` and `discreteSample`, this should provide what you are looking for.

Cheers, Rüdiger

**Reply by JasonOttUW** [reply]
Thank you so much, just what I needed (the unimodalODF)!

**Comment by makeabhishek** [comment]
Hi I'm trying to generate random orientations for statistically isotropic material properties. How can I generate that?

Thanks and Regards
Abhishek

**Reply by nyyssont** [reply]
Please see the section "Random orientations":

https://mtex-toolbox.github.io/OrientationDefinition.html
